OISF/suricata
# Pcre2 3194 v3

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/3194

Describe changes:
- Use PCRE2 instead of pcre for transform pcrexform
- qa : remove some obsolete files
- pcrexform : checks that we have exactly one substring capture in a signature

This draft PR is about testing PCRE2 support by the CI
After that, the goal is to convert detect-pcre.c to PCRE2

suricata-verify-pr: 480

https://github.com/OISF/suricata-verify/pull/480
